Data/ODBC/ODBC.make

```diff
@@ -12,6 +12,16 @@ POCO_ODBC_INCLUDE = /opt/homebrew/include
 endif
 endif
 
+ifndef POCO_MSSQL_INCLUDE
+ifeq (0, $(shell test -e /usr/include/msodbcsql.h; echo $$?))
+POCO_MSSQL_INCLUDE = /usr/include
+else ifeq (0, $(shell test -e /opt/microsoft/msodbcsql18/include/msodbcsql.h; echo $$?))
+POCO_MSSQL_INCLUDE = /opt/microsoft/msodbcsql18/include
+else ifeq (0, $(shell test -e /opt/homebrew/include/msodbcsql18/msodbcsql.h; echo $$?))
+POCO_MSSQL_INCLUDE = /opt/homebrew/include/msodbcsql18
+endif
+endif
+
```

Comment:
Shall these settings be added to CMake also?

Reply:
Yes, in order for SQL Server big strings to work, at compile time the path to `mssqlodbc.h` should be in compiler include directories list. On some systems I've seen that header in `/usr/include`, but mostly it's not there it seems. We try our best to find it automatically, but we also provide the way to explicitly specify it:

https://github.com/pocoproject/poco/blob/a97fd9eaa5c028350c6044c8c648ccc19eac117d/configure#L212-L213

https://github.com/pocoproject/poco/blob/a97fd9eaa5c028350c6044c8c648ccc19eac117d/configure#L384-L385

At this time, we only need [SQL_SS_LENGTH_UNLIMITED](https://github.com/pocoproject/poco/blob/a97fd9eaa5c028350c6044c8c648ccc19eac117d/Data/ODBC/src/Binder.cpp#L170), but even that is a symbol not guaranteed to have the same numeric value forever, and there may be more such things in the future (there are always some nonstandard specifics that leak through the abstraction)